### ISSUE TITLE
Updating a date field

### ISSUE THREAD

**Issue by lookfirst** [Airtable/airtable.js#125]
I'd love for the ability to update a Date field, by just passing in a JS Date object.

**Comment by lookfirst** [comment]
Ah, just found typecast.

**Comment by lookfirst** [comment]
Re-opened, typecast, which is also missing from the typescript declarations, doesn't work.

**Comment by reelmatt** [comment]
@lookfirst Following the examples in the [documentation for a base's API](https://airtable.com/api), I got the creation and updating of a single record to work for a JS Date object. The typecast parameter seems to be required (as I got a 422 error otherwise). The JS Date worked for me passing through the object as a whole, and calling the `toISOString()` function on the date object.

```javascript
// create new JS Date object
let now = new Date();
let base = new Airtable({apiKey: 'YOUR_API_KEY'}).base('YOUR_BASE_ID');

// update a specific Airtable record; also worked for me with base().create()
base('YOUR_TABLE_NAME').update('YOUR_RECORD_ID', {
    "Date": now    // alternatively, now.toISOString() also worked for me
}, {typecast: true}, function (err, record) {
    if (err) {
        console.error(err);
        return;
    }
    console.log(record);
});
```

However, I was not able to get the calls to succeed, with or without the `{typecast: true}` parameter, when trying to perform pass in an array of record objects, compared to a single record. The call resulted in a 422 `"INVALID_REQUEST_UNKNOWN"` error; not sure why that's failing when a single record succeeds. I actually haven't run into that problem as most of my code will loop through an array of records and make calls to update/create individually. (Side note: I believe passing in multiple records is a relatively new, or even beta, feature, so that may explain some bugs there.)

Hope this helps!
